app/javascript/mastodon/locales/en.json

```diff
@@ -366,6 +366,8 @@
   "load_pending": "{count, plural, one {# new item} other {# new items}}",
   "loading_indicator.label": "Loading...",
   "media_gallery.toggle_visible": "{number, plural, one {Hide image} other {Hide images}}",
+  "missing_indicator.label": "Not found",
+  "missing_indicator.sublabel": "This resource could not be found",
```

Comment:
These seem to have been mistakenly re-added for some reason.

Reply:
I had a similar thing happen on another PR, not sure where these are coming from. I had rebuilt first but the reappeared for me too, but not on CI

Reply:
Maybe leftover from `build/messages`?

Reply:
Looks like I needed to clean build and run `RAILS_ENV=development ./bin/rails assets:precompile` again before the other Yarn calls to clear those

Reply:
removed it manually, when I tried to do it with build it kept making messages for some changes I had on a different branch